kubernetes/kserve/kserve_wrapper/TorchserveModel.py

```diff
@@ -48,77 +69,52 @@ def __init__(self, name, inference_address, management_address, model_dir):
-        logging.info("PREDICTOR_HOST : %s", self.predictor_host)
-        headers = {"Content-Type": "application/json; charset=UTF-8"}
-        response = await self._http_client.fetch(
-            PREDICTOR_URL_FORMAT.format(self.predictor_host, self.name),
-            method="POST",
-            request_timeout=self.timeout,
-            headers=headers,
-            body=json.dumps(request),
-        )
-
-        if response.code != 200:
-            raise tornado.web.HTTPError(status_code=response.code, reason=response.body)
-        return json.loads(response.body)
-
-    async def explain(self, request: Dict) -> Dict:
-        """The predict method is called when we hit the explain endpoint and handles the
-        explain request and response from the Torchserve side and passes it on to the
-        KServe side.
-
-        Args:
-            request (Dict): Input request from the http client side.
+        if self.protocol == PredictorProtocol.GRPC_V2.value:
+            self.predictor_host = grpc_inference_address
 
-        Raises:
-            NotImplementedError: If the predictor host on the KServe side is not
-                                 available.
+        logging.info("Predict URL set to %s", self.predictor_host)
+        logging.info("Explain URL set to %s", self.explainer_host)
 
-            tornado.web.HTTPError: If there is a bad response from the http client.
-
-        Returns:
-            Dict: The Response from the input from the explain endpoint.
-        """
-        if self.explainer_host is None:
-            raise NotImplementedError
-        logging.info("kfmodel explain request is %s", json.dumps(request))
-        logging.info("EXPLAINER_HOST : %s", self.explainer_host)
-        headers = {"Content-Type": "application/json; charset=UTF-8"}
-        response = await self._http_client.fetch(
-            EXPLAINER_URL_FORMAT.format(self.explainer_host, self.name),
-            method="POST",
-            request_timeout=self.timeout,
-            headers=headers,
-            body=json.dumps(request),
-        )
-        if response.code != 200:
-            raise tornado.web.HTTPError(status_code=response.code, reason=response.body)
-        return json.loads(response.body)
+    def grpc_client(self):
+        if self._grpc_client_stub is None:
+            self.channel = grpc.aio.insecure_channel(self.predictor_host)
+            self.grpc_client_stub = inference_pb2_grpc.InferenceAPIsServiceStub(
+                self.channel
+            )
+        return self.grpc_client_stub
+
+    async def _grpc_predict(
+        self,
+        payload: Union[ModelInferRequest, InferRequest],
+        headers: Dict[str, str] = None,
+    ) -> ModelInferResponse:
+        payload = to_ts_grpc(payload)
+        grpc_stub = self.grpc_client()
+        async_result = await grpc_stub.Predictions(payload)
+        return async_result
+
+    def postprocess(
```

Comment:
I don't see this function is called. Could you add some comments (eg. which function call this one)?